### PR TITLE
Update aws-ecs-deployment.md

### PR DESCRIPTION
**Pull request by iamtodor**
Extend documentation for AWS ECS Installation https://github.com/amundsen-io/amundsen/issues/875

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely, including a title prefix.
- [ ] PR includes a summary of changes.
- [ ] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by feng-tao** [comment]
could you set your DCO?

File: docs/installation-aws-ecs/aws-ecs-deployment.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ $ export AWS_PROFILE=profilename
 
 For the purpose of this instruction we used the [tutorial](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-ec2.html#ECS_CLI_tutorial_compose_create) on AWS documentation
 
+Enter the cloned directory:
+
+```
+cd amundsen/docs/installation-aws-ecs
```
Comment (feng-tao) [inline docs/installation-aws-ecs/aws-ecs-deployment.md:23]: all the comment mentioned `~/<your-path-to-cloned-repo>/amundsenfrontendlibrary/docs/instalation-aws-ecs` , maybe we could change those comments (this doc was contributed back then we don't have the amundsen repo.

**Comment by iamtodor** [comment]
@feng-tao Hello Tao,
I appreciate your answer and your help :slightly_smiling_face:
Could you please specify what is DCO? Cause I haven't met such a definition yet

File: docs/installation-aws-ecs/aws-ecs-deployment.md (hunk 1 above)
Comment (iamtodor) [reply docs/installation-aws-ecs/aws-ecs-deployment.md:23]: @feng-tao totally agree

**Comment by iamtodor** [comment]
@feng-tao Hi Tao,
DCO has been successfully set

**Review by feng-tao — APPROVED** [review]

**Pull request merged**